ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -419,7 +419,9 @@ class TFBuffer : public TensorBuffer {
   void* data_;
   size_t len_;
 
+#if (TF_MAJOR_VERSION == 1 && TF_MINOR_VERSION < 13)
```

Comment:
Since before 1.13, the method `data ()` was virtual, I set the condition like this.